src/helpers/EmailNotificationAPI.js

```diff
@@ -0,0 +1,46 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
```

Comment:
JSdoc is required for commenting. 
See sample here [Js doc](http://usejsdoc.org/)
Nice work though

Reply:
JSdocs have been added